### PR TITLE
Add post-installation instructions

### PR DESCRIPTION
**Pull request by jamesbt365**
Should make it less confusing for linux users

should also do the other half of stopping the person from #71 for trying to add stuff that is somewhere and then not add it to the other instructions.

### CODE REVIEW

File: OpenTabletDriver.Web/Views/Wiki/Install/Linux.cshtml [hunk 1]
```diff
@@ -124,3 +124,9 @@
         </codeblock>
     </li>
 </ol>
+<hr/>
+
+<h3 class="wiki-nav-item" id="post-install">
+    Post-Installation
+</h3>
+<p>After installation, you may be required to look at the console of the OpenTabletDriver to find any problems, then follow the <a href="/Wiki/FAQ/Linux">FAQ</a> before your tablet works.</p>
```
Comment (InfinityGhost) [inline OpenTabletDriver.Web/Views/Wiki/Install/Linux.cshtml:132]: Not sure I like this, needs reworded in a shorter manner.

**Review by InfinityGhost — CHANGES_REQUESTED** [review]

File: OpenTabletDriver.Web/Views/Wiki/Install/Linux.cshtml (hunk 1 above)
Comment (vedattt) [reply OpenTabletDriver.Web/Views/Wiki/Install/Linux.cshtml:132]:
```suggestion
<p>You're advised to look at the console tab on OpenTabletDriver UX to catch any problems. Then if necessary, read through the <a href="/Wiki/FAQ/Linux">FAQ</a>.</p>
```
Is this better?

Comment (jamesbt365) [reply OpenTabletDriver.Web/Views/Wiki/Install/Linux.cshtml]: Resolved.

File: OpenTabletDriver.Web/Views/Wiki/Install/Linux.cshtml [hunk 2]
```diff
@@ -124,3 +124,9 @@
         </codeblock>
     </li>
 </ol>
+<hr/>
+
+<h3 class="wiki-nav-item" id="post-install">
+    Post-Installation
+</h3>
+<p>You're advised to look at the console tab on OpenTabletDriver UX to catch any problems. Then if necessary, read through the <a href="/Wiki/FAQ/Linux">FAQ</a>.</p>
```
Comment (InfinityGhost) [inline OpenTabletDriver.Web/Views/Wiki/Install/Linux.cshtml:132]:
```suggestion
<p>You're advised to look at the console tab on OpenTabletDriver UX to catch any problems. If necessary, read through the <a href="/Wiki/FAQ/Linux">FAQ</a>.</p>
```

**Review by InfinityGhost — CHANGES_REQUESTED** [review]

**Review by InfinityGhost — APPROVED** [review]

**Pull request merged**